src/parse-stream.js

```diff
@@ -363,11 +363,53 @@ export default class ParseStream extends Stream {
       this.trigger('data', event);
       return;
     }
+    match = (/^#EXT-X-CUE-OUT-CONT:?(.*)?$/).exec(line);
```

Comment:
These three match-blocks are almost exactly the same. I came up with:

```javascript
match = (/^#EXT-X-(CUE-(?:(?:IN)|(?:OUT-CONT)|(?:OUT)))(?::(.*))?$/).exec(line);
if (match && match.length >= 2) {
  event = {
    type: 'tag',
    tagType: match[1].toLowerCase()
  };
  if (match[1]) {
    event.data = match[2];
  } else {
    event.data = '';
  }
  this.trigger('data', event);
  return;
}
```

But I'm not sure that is better even if it is more compact and less redundant.